deployment/operator/README.md

```diff
@@ -12,7 +12,7 @@ nri-plugin specified in the NriPluginDeployment.
 
 Build the operator image and push it to some registry
 ```shell
-make -C deployment/operator docker-build docker-push IMG="my-registry.com/nri-plugins-operator:unstable"
+make -C deployment/operator docker-build docker-push IMAGE="my-registry.com/nri-plugins-operator:unstable"
```

Comment:
nitpick: just want to note (so we can try to be more careful in the future), that this is an unrelated documentation fix change, which is not in its own commit and neither mentioned in the commit message nor in the PR description.

Reply:
Thanks for the comment @klihub . I will keep that in mind in the future PRs.